adserver/admin.py

```diff
@@ -670,14 +670,14 @@ def action_create_draft_invoice(self, request, queryset):
             )
 
         # https://stripe.com/docs/api/invoices/create
+        description = "Thanks for your business!"
```

Comment:
Ha, I never noticed this before :)

Reply:
I usually fill out the memo line to reference a PO or if this is "invoice 1 of 3", etc.